src/python/src/grpc/_adapter/_c/module.c

```diff
@@ -31,21 +31,31 @@
  *
  */
 
-#ifndef _ADAPTER__SERVER_H_
```

Comment:
"src/python/src/grpc/_adapter/_server.h → src/python/src/grpc/_adapter/_c/module.c" is a funny move to make, and makes for a funny diff - how easily could you change this file to being a fork of something more sensible (_c.c?)?